assets/js/Ioda/pages/dashboard/DashboardTab.js

```diff
@@ -133,81 +133,82 @@ class DashboardTab extends Component {
                       text={tooltipDashboardHeadingText}
                     />
                     <div className="col" />
-                    {this.props.type !== asn.type && (
+                    {type !== asn.type && (
                       <>
                         <div className="text-lg mr-4">
-                          {this.props.tabCurrentView === "timeSeries"
+                          {tabCurrentView === "timeSeries"
                             ? viewTitleChart
                             : viewTitleMap}
                         </div>
                         <Button
                           type="primary"
-                          onClick={this.props.handleTabChangeViewButton}
+                          onClick={handleTabChangeViewButton}
                           icon={
-                            this.props.tabCurrentView === "timeSeries" ? (
+                            tabCurrentView === "timeSeries" ? (
                               <GlobalOutlined />
                             ) : (
                               <AreaChartOutlined />
                             )
                           }
                           aria-label={
-                            this.props.tabCurrentView === "timeSeries"
+                            tabCurrentView === "timeSeries"
                               ? viewChangeIconAltTextHts
                               : viewChangeIconAltTextMap
                           }
                         />
                       </>
                     )}
                   </div>
-                  {this.props.type !== "asn" ? (
+                  {type !== "asn" ? (
                     <div
                       className="dashboard__tab-map"
                       style={
-                        this.props.tabCurrentView === "map"
+                        tabCurrentView === "map"
                           ? { display: "block" }
                           : { display: "none" }
                       }
                     >
-                      {this.props.topoData &&
-                      this.props.summaryDataRaw &&
-                      this.props.totalOutages &&
-                      this.props.topoScores
-                        ? this.genMap()
+                      {topoData &&
+                      summaryDataRaw &&
+                      totalOutages &&
+                      topoScores
+                        ? genMap()
```

Comment:
Related to my previous comment about `genMap`, this could be replaced with:

```
{topoData &&
 summaryDataRaw &&
 totalOutage &&
 topoScore &&
 (
    <TopoMap
        topoData={topoData}
        scores={topoScores}
        handleEntityShapeClick={(entity) => handleEntityShapeClick(entity)}
        entityType={activeTabType?.toLowerCase()}
     />
}